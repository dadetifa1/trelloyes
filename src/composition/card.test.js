import React from 'react';
import ReactDOM from 'react-dom';
import cardClass from './card';
import renderer from 'react-test-renderer';


describe('Card component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<cardClass />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<cardClass title="Test title" content="Test Content"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

  it('renders the UI as expected with no prop values', () => {
    const tree = renderer
      .create(<cardClass title="" content=""/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
}); 