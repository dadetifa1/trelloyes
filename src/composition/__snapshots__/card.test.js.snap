// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the UI as expected 1`] = `
<cardClass
  content="Test Content"
  title="Test title"
/>
`;

exports[`Card component renders the UI as expected with no unreads 1`] = `
<cardClass
  content=""
  title=""
/>
`;
